### PR TITLE
change: enable release build in new build system

### PR DESCRIPTION
**Pull request by ChuyangDeng**
*Issue #, if available:*

*Description of changes:*
enable Chainer build release in new build system

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by jesterhazy** [comment]
### AWS CodeBuild CI Report

* Result: SUCCEEDED
* [Build Logs](https://2hpflh3jba.execute-api.us-west-2.amazonaws.com/Prod/buildlogs?key=f0463573-3eef-47e4-be2f-d8afc4e591e7%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

File: buildspec-release.yml [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+      - |
+        if has-matching-changes "test/" "tests/" "src/*.py" "docker/*" "buildspec.yml"; then
+          pytest test/integration/sagemaker --region $AWS_DEFAULT_REGION --docker-base-name $ECR_REPO --aws-id $ACCOUNT --tag $GPU_TAG --instance-type $GPU_INSTANCE_TYPE
+        else
+          echo "skipping gpu sagemaker tests"
+        fi
+
+      - |
+        echo '[{
+          "repository": "sagemaker-pytorch",
```
Comment (mvsusp) [inline buildspec-release.yml]: sagemaker-chainer

File: buildspec-release.yml [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+            "dest": ["5.0.0-cpu-py2", "5.0-cpu-py2", "5.0.0-cpu-py2-'${CODEBUILD_BUILD_ID#*:}'"]
+          },{
+            "source": "5.0.0-cpu-py3",
+            "dest": ["5.0.0-cpu-py3", "5.0-cpu-py3", "5.0.0-cpu-py3-'${CODEBUILD_BUILD_ID#*:}'"]
+          },{
+            "source": "5.0.0-gpu-py2",
+            "dest": ["5.0.0-gpu-py2", "5.0-gpu-py2", "5.0.0-gpu-py2-'${CODEBUILD_BUILD_ID#*:}'"]
+          },{
+            "source": "5.0.0-gpu-py3",
+            "dest": ["5.0.0-gpu-py3", "5.0-gpu-py3", "5.0.0-gpu-py3-'${CODEBUILD_BUILD_ID#*:}'"]
```
Comment (mvsusp) [inline buildspec-release.yml]: Is 5.0.0 the latest version?

Comment (ChuyangDeng) [reply buildspec-release.yml]: 5.0.0 is the latest version.

**Comment by jesterhazy** [comment]
### AWS CodeBuild CI Report

* Result: SUCCEEDED
* [Build Logs](https://2hpflh3jba.execute-api.us-west-2.amazonaws.com/Prod/buildlogs?key=675b7557-8165-42a9-a9d4-9d53c26ae9e1%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Pull request merged**